packages/dashboard/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@uifabric/dashboard",
-  "version": "0.58.2",
+  "version": "0.59.0",
```

Comment:
Why is this package back??? The entire dashboard package is supposed to be gone!